cmd/config/config.go

```diff
@@ -311,23 +269,46 @@ func writeConfig(
 	return nil
 }
 
-func newErr(flag string) error {
+// writeConfig writes the values in config to the config file based on the filter function.
+func writeConfig2(
```

Comment:
This is the same as `writeConfig` but takes a `Config` instead of a `ConfigFile`.

Reply:
could the other one be renamed `writeConfigFile` and this one is `writeConfig` ?